src/iOS/Services/iOSPushNotificationService.cs

```diff
@@ -84,14 +85,19 @@ public void SendLocalNotification(string title, string message, BaseNotification
             var content = new UNMutableNotificationContent()
             {
                 Title = title,
-                Body = message
+                Body = message,
+                CategoryIdentifier = Constants.iOSNotificationCategoryId
             };
 
             if (data != null)
             {
                 content.UserInfo = NSDictionary.FromObjectAndKey(NSData.FromString(JsonConvert.SerializeObject(data), NSStringEncoding.UTF8), new NSString(Constants.NotificationData));
             }
 
+            var actions = new UNNotificationAction[] { UNNotificationAction.FromIdentifier(Constants.iOSNotificationClearActionId, AppResources.Clear, UNNotificationActionOptions.Foreground) };
+            var category = UNNotificationCategory.FromIdentifier(Constants.iOSNotificationCategoryId, actions, new string[] { }, UNNotificationCategoryOptions.CustomDismissAction);
+            UNUserNotificationCenter.Current.SetNotificationCategories(new NSSet<UNNotificationCategory>(new UNNotificationCategory[] { category }));
```

Comment:
The `NSSet` takes a `params UNNotificationCategory[] objs` as a parameter so there's no need to create an array there, i.e. directly use:
```c#
UNUserNotificationCenter.Current.SetNotificationCategories(new NSSet<UNNotificationCategory>(category));
```